src/Microsoft.Extensions.WebEncoders/EncoderServiceCollectionExtensions.cs

```diff
@@ -17,14 +20,23 @@ public static class EncoderServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/>.</param>
         /// <returns>The <see cref="IServiceCollection"/> instance after the encoders have been added.</returns>
-        public static IServiceCollection AddWebEncoders(this IServiceCollection services)
+        public static void AddWebEncoders(this IServiceCollection services)
```

Comment:
@HaoK - it makes the other builders and their methods less discoverable. This was discussed up at the time and we wanted to optimize for discoverability.